core/config/v2/types.go

```diff
@@ -31,8 +31,8 @@ var ErrUnsupported = errors.New("unsupported with config v2")
 // Core holds the core configuration. See chainlink.Config for more information.
 type Core struct {
 	// General/misc
-	AppID               uuid.UUID `toml:"-"`
-	DevMode             bool      `toml:"-"`
+	AppID               uuid.UUID `toml:"-"` // random or test
```

Comment:
What is the meaning of this comment?

Reply:
I was noting where the non-toml field values come from since these are the only exceptions. `AppID` is randomized or sometimes explicitly set for a test. `DevMode` is set from the environment.